hil/ext/switches/mock.py

```diff
@@ -56,6 +56,11 @@ def validate(kwargs):
             'password': basestring,
         }).validate(kwargs)
 
+    @staticmethod
+    def validate_port_name(port):
+        """All port names are valid for the mock switch"""
+        pass
```

Comment:
Cool. Since all switches have something like slot#/port# or slot/chassis/port format for port numbers, the valid port name for a mock switch is like 1/0/1 or 1/1. What do you think @SahilTikale @zenhack ?